### PR TITLE
chore: implement new sentry sdk and remove raven dependency

### LINKED ISSUE #5837: Integrate and verify new sentry SDK
Update sentry SDK and verify it works

### PR DESCRIPTION
**Pull request by uds5501**
Fixes #5837 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
This implements new `sentry-sdk` and removes the legacy `raven[flask]` dependency

### CODE REVIEW

**Comment by uds5501** [comment]
@iamareebjamal review please

File: requirements/common.txt [hunk 1]
```diff
@@ -48,7 +48,7 @@ wtforms~=2.2
 flask-admin~=1.5
 google-compute-engine~=2.8
 factory_boy~=2.12
-raven[flask]~=6.10
+sentry-sdk[flask]
```
Comment (iamareebjamal) [inline requirements/common.txt]: Add version number

**Comment by iamareebjamal** [comment]
Please verify on a sentry project that it is working

File: requirements/common.txt (hunk 1 above)
Comment (uds5501) [reply requirements/common.txt]: @iamareebjamal  okay

**Comment by uds5501** [comment]
@iamareebjamal I have run it on my our project itself, I am attaching the logs :

image

**Comment by iamareebjamal** [comment]
`on my our project` - my or our?

**Comment by iamareebjamal** [comment]
Remove WIP status when it's ready for review

**Comment by uds5501** [comment]
@iamareebjamal on my own project (local repo) :laughing:

**Pull request merged**